lisp_ease/src/queries/grasping.lisp

```diff
@@ -0,0 +1,298 @@
+                                                              grasp-transform)
+  (cram-tf:translate-transform-stamped grasp-transform :x-offset (- *bowl-pregrasp-z-offset*)))
+;------------------------------------------------------------------------------------------------
+; CUP -------------------------------------------------------------------------------------------
+;------------------------------------------------------------------------------------------------
+
+(defmethod get-object-type-to-gripper-transform ((object-type (eql :ba-cup))
+                                                 object-name
+                                                 arm
+                                                 (grasp (eql :human-grasp)))
+  (print "GRASPING STUFF LIKE A HUMAN.")
+  (let* (transf
+         end-transf)
+
+    ;; transf. from Map to Obj?
+    (setq transf
+          (cl-tf:transform*
+            (cl-tf:transform-inv
+               (make-poses "?PoseObjStart"))
+           (make-poses "?PoseHandStart")
+           (human-to-robot-hand-transform)))
+    
+    (print "***human-grasp***")
+    (setf end-transf
+          (cl-tf:transform->transform-stamped
+           (roslisp-utilities:rosify-underscores-lisp-name object-name)
+           (ecase arm
+             (:left cram-tf:*robot-left-tool-frame*)
+             (:right cram-tf:*robot-right-tool-frame*))
+           0.0
+           transf))
+
+    (print "------------------------------")
+    (print "THIS IS THE END TRANSFORM:")
+    (print "------------------------------")
+    (print end-transf)
+    end-transf))
+
+(defmethod get-object-type-to-gripper-pregrasp-transform ((object-type (eql :ba-cup))
+                                                          object-name
+                                                          arm
+                                                          (grasp (eql :human-grasp))
+                                                          grasp-transform)
+  (cram-tf:translate-transform-stamped grasp-transform :x-offset (- *cup-pregrasp-xy-offset*)
+                                                       :z-offset *lift-z-offset*))
+(defmethod get-object-type-to-gripper-2nd-pregrasp-transform ((object-type (eql :ba-cup))
+                                                              object-name
+                                                              arm
+                                                              (grasp (eql :human-grasp))
+                                                              grasp-transform)
+  (cram-tf:translate-transform-stamped grasp-transform :x-offset (- *cup-pregrasp-xy-offset*)))
+
+
+;;-----------------------------------------------------------------------------
+;; FORK
+;;----------------------------------------------------------------------------
+(defmethod get-object-type-to-gripper-transform ((object-type (eql :ba-fork))
```

Comment:
Get rid of all ``get-object-type-to-gripper-transform`` functions in this file.
Keep one, instead of ``(object-type (eql :object-type))`` have a general one: simply ``object-type``.